Task2/Project/DataControl/ViewModel/ApplicationViewModel.cs

```diff
@@ -75,16 +86,17 @@ public ApplicationViewModel(IFileService fileService, IDialogService dialogServi
 
             addVertex = new RelayCommand(AddVertexMethod);
             deleteShape = new RelayCommand(DeleteShapeMethod, CanDeleteShapeAction);
-
-            throw new System.NotImplementedException();
         }
 
         // PROPERTIES
+        /// <summary>
+        /// Returns name of file.
+        /// </summary>
         public string FileName
         {
             get
             {
-                throw new System.NotImplementedException();
+                return currentFileName;

```

Comment:
you do not want to show all path to file, just file name

Reply:
Okey